### PR TITLE
Fix connected state

### LINKED ISSUE #8: Wallet will never get `connected`
Hey, there has to be something wrong on undesired.

From the apparent point of view the wallet will never get connected.
Looking at the code - it will never go to line https://github.com/aeternity/aepp-boilerplate-vue/blob/03c9eb6285da25e7ad2c86b7888007ac2cf7e56f/src/utils/aeternity/wallet.js#L69 and `walletStatus` variable will never be assigned with `connected` value. It will get back to `null` actually

Another way howto analyze this is by adding watcher to `App.vue`

```
    watch(walletStatus,
      async (newStatus, oldStatus) => {
        console.log('oldStatus', oldStatus, '=> newStatus', newStatus)
      })
```
This will be the result
obrazek

As from intuition and code readability I would expect here
`<div v-if="address && walletStatus === 'connected'" class="wallet-info">`
but in the codebase there is 
`<div v-if="address && !walletStatus" class="wallet-info">`

https://github.com/aeternity/aepp-boilerplate-vue/blob/03c9eb6285da25e7ad2c86b7888007ac2cf7e56f/src/components/WalletInfo.vue#L5
Meaning-wise `walletStatus = null` means that there is no information about wallet connection

Seems like there is some waiting for finishing connection missing.

I guess the solution is variation of this https://github.com/aeternity/aepp-sdk-js/pull/1501#issuecomment-1134359307

Please fix this soon as possible as I am using this to build Multisig app on this. It is blocking the clear way of implementation.
Thank you

### PR DESCRIPTION
**Pull request by janmichek**
fixes https://github.com/aeternity/aepp-boilerplate-vue/issues/8

Now It is branched from master, so it can be directly reviewed and merged.

https://user-images.githubusercontent.com/15363559/181022859-0af1cc6c-c6f2-46b2-bc7d-b1005882d028.mp4

### CODE REVIEW

File: src/components/WalletInfo.vue [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-      <strong>Address :: </strong> <span class="wallet-address">{{ address }}</span>
-      <br />
-      <strong>Balance :: </strong> <span class="wallet-balance">{{ balance }}</span>
-    </div>
+    <template v-if="walletStatus">
+      <strong>Wallet Status :: </strong> {{ walletStatus }}
+      <br/>
+    </template>
+
+    <template v-if="address && walletStatus === 'connected'">
```
Comment (janmichek) [inline src/components/WalletInfo.vue:8]: now this condition make sense

File: src/components/WalletInfo.vue [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+      <template v-if="activeWallet">
+        <strong>Provider :: </strong> {{ activeWallet.info.name }}
```
Comment (janmichek) [inline src/components/WalletInfo.vue:10]: also simplified the markup a bit

File: src/components/WalletInfo.vue [hunk 3]
```diff
@@ -22,38 +25,27 @@
...
-export default defineComponent({
-  name: 'WalletInfo',
-  setup () {
-    const { address, balance, walletStatus, activeWallet } = toRefs(aeWallet)
-
-    return { activeWallet, address, balance, walletStatus }
-  }
-})
+// probably needs deps upgrading deps
+// eslint-disable-next-line
```
Comment (janmichek) [inline src/components/WalletInfo.vue:33]: this acts kinda strange, hopefully will be resolved with https://github.com/aeternity/aepp-boilerplate-vue/issues/13

File: src/utils/aeternity/wallet.js [hunk 4]
```diff
@@ -115,14 +109,29 @@ export const aeFetchWalletInfo = async (sdk)  => {
     address.value = await sdk.address()
 
     balance.value = await sdk.getBalance(address.value, {
-      format: AmountFormatter.AE_AMOUNT_FORMATS.AE
+      format: AE_AMOUNT_FORMATS.AE,
     })
-
-    walletStatus.value = null
+    walletStatus.value = 'connected'
```
Comment (janmichek) [inline src/utils/aeternity/wallet.js:112]: This makes much more sense

**Comment by marc0olo** [comment]
just ignore the comment here as were discussing in #11

File: src/utils/aeternity/client.js [hunk 5]
```diff
@@ -20,8 +20,8 @@ export const aeClient = reactive({
  * This client can not sign transactions that require funds (everything except static contract calls)
  * @returns {Promise<boolean>}
  */
-export const aeInitClient = async () => {
-  const { isConnected, isConnecting, isStatic } = toRefs(aeClient)
+export const initClient = async () => {
```
Comment (janmichek) [inline src/utils/aeternity/client.js:23]: I have removed these prefixing as I saw no benefit in it, only bloating code and worse readiness

**Comment by marc0olo** [comment]
@janmichek is that PR still relevant? code-wise it seems fine to me. anything to check from UI perspective? what changed compared to #11 ?

**Comment by janmichek** [comment]
> @janmichek is that PR still relevant? code-wise it seems fine to me. anything to check from UI perspective? 

I have minimized the markup of WalletInfo

> what changed compared to https://github.com/aeternity/aepp-boilerplate-vue/pull/11 ?

You can see the changes in Files Changed tab.
Just asking you to do standard code review, nothing special.
Please look at the code and do some basic app walkthrough.

**Review by marc0olo — APPROVED** [review]
lgtm

**Pull request merged**